Framework/PythonInterface/test/CMakeLists.txt

```diff
@@ -52,3 +52,4 @@ endfunction()
 add_subdirectory(python)
 add_subdirectory(cpp)
 add_subdirectory(testhelpers)
+add_dependencies(FrameworkTests Framework)
```

Comment:
Is there a reason this is done here and not in `Framework/CMakeLists.txt`?

Reply:
Good point. Nice catch. Yes that makes more sense to do in `Framework/CMakeLists.txt`.

Reply:
Force pushed the move to `Framework/CMakeLists.txt`